ai2thor/tests/data/metadata-schema.json

```diff
@@ -1,662 +1,595 @@
-                                ]
-                            }
-                        },
-                        "required": [
-                            "center",
-                            "cornerPoints",
-                            "size"
-                        ]
-                    },
-                    "objectOrientedBoundingBox": {
-                        "anyOf": [
-                            {
-                                "type": "null"
-                            },
-                            {
-                                "type": "object",
-                                "properties": {
-                                    "cornerPoints": {
-                                        "type": "array",
-                                        "items": {
-                                            "type": "array",
-                                            "items": {
-                                                "type": "number"
-                                            }
-                                        }
-                                    }
-                                },
-                                "required": [
-                                    "cornerPoints"
-                                ]
-                            }
-                        ]
-                    }
-                },
-                "required": [
-                    "ObjectTemperature",
-                    "axisAlignedBoundingBox",
-                    "breakable",
-                    "canBeUsedUp",
-                    "canChangeTempToCold",
-                    "canChangeTempToHot",
-                    "canFillWithLiquid",
-                    "cookable",
-                    "dirtyable",
-                    "distance",
-                    "isBroken",
-                    "isCooked",
-                    "isDirty",
-                    "isFilledWithLiquid",
-                    "isMoving",
-                    "isOpen",
-                    "isPickedUp",
-                    "isSliced",
-                    "isToggled",
-                    "isUsedUp",
-                    "mass",
-                    "moveable",
-                    "name",
-                    "objectId",
-                    "objectOrientedBoundingBox",
-                    "objectType",
-                    "obstructed",
-                    "openable",
-                    "parentReceptacles",
-                    "pickupable",
-                    "position",
-                    "receptacle",
-                    "receptacleObjectIds",
-                    "rotation",
-                    "salientMaterials",
-                    "sliceable",
-                    "toggleable",
-                    "visible"
-                ]
-            }
-        },
-        "isSceneAtRest": {
-            "type": "boolean"
-        },
-        "agent": {
+  "$schema": "http://json-schema.org/draft-07/schema",
```

Comment:
Applied autoformatting to this file. The only thing that actually changes is "hand" is now "heldObjectPose".